Added GLB import information and new asset tasks (Draco + FBX folder) (#493)
playcanvas/developer.playcanvas.com: content/en/user-manual/assets/models/building.md

Building art and animations for PlayCanvas can be done using almost any of the many 3D modeling programs available. For example: Blender, SketchUp, Autodesk 3D Studio Max or Autodesk Maya.

PlayCanvas is designed to import content most faithfully via the open source glTF binary (GLB) format so, in general, if your modeling application supports GLB, PlayCanvas supports it too.

The import of GLBs supports the following glTF features for incredibly fast scene creation:

- Node hierarchy
- Standard material
- Ratified material extensions
- Animated skeletons and skinning data
- Morph targets
- Cameras
- Draco compressed models

> - Light punctual support

Reviewer: ```suggestion
- Punctual light support
```

> As such, to target PlayCanvas, you do not have to use any special art tools plugins and there are no PlayCanvas-specific workflows you must follow.

Reviewer: ```suggestion
As such, to target PlayCanvas, you do not have to use any special art tool plugins and there are no PlayCanvas-specific workflows you must follow.
```

![GLB Import Example][14]

(Model by Loïc Norgeot and mosquito scan by Geoffrey Marchal for [Sketchfab][13] licensed under [CC BY 4.0][12]).

> PlayCanvas also supports importing of FBX mode which is another common interchange format. However, it doesn't retain material properties and you will need to configure them after import.

Reviewer: ```suggestion
PlayCanvas also supports the importing of FBX files which is another common interchange format. However, FBX materials are not imported as faithfully as glTF materials and might require some editing after import.
```

[Blender][4] is a fantastic free tool for 3D modeling and it is perfect for generating 3D assets for PlayCanvas.

Blender supports exporting to GLB, FBX and COLLADA but the built-in exporters do have some limitations.

### **Map Types**

> 

Author: I don't know if the rest of the page is still accurate so left as is
